gradlew

```diff
@@ -112,19 +114,6 @@ case "$( uname )" in                #(
   NONSTOP* )        nonstop=true ;;
 esac
 
-
-# Loop in case we encounter an error.
```

Comment:
idk why its making this change.


@timtebeek 
can you give me some context here plz? 

the doc does not help me in this case. 

- https://docs.gradle.org/current/userguide/best_practices_general.html

Reply:
We take the gradlew scripts as provided by gradle and apply those. You'll notice the reference script does not contain this block:
https://github.com/gradle/gradle/blob/master/platforms/jvm/plugins-application/src/main/resources/org/gradle/api/internal/plugins/unixStartScript.txt